### ISSUE TITLE
Get Lexis-Nexis section ID more reliably

### ISSUE THREAD

**Issue by waldoj** [statedecoded/statedecoded#640]
The Lexis-Nexis XSL is getting the section identifier from a problematic source. e.g.:
```php
<anchor id="_64.1-01" />
<heading>
   <title>Repealed</title>
   <desig>§§ 64.1-01 through 64.1-206.8.</desig>
</heading>
```
Our source is `desig`, not `anchor` `id`. This sample XML demonstrates the problem with that—the designator is not necessarily an actual section number—sometimes it represents a list or a range of them. `anchor` `id` is less accurate, but far more appropriate for our purposes. Further, it is identified in the XML as _the_ ID, and, as such, should be used.

**Comment by krues8dr** [comment]
I punt on this one too, but in the opposite way so it still shows the original section numbers to the user on the frontend.

This would also be a good place to use the new permalinks system to have aliases for several url options.

**Comment by waldoj** [comment]
The problem is that's not the outcome. The whole system operates under the assumption that section identifiers are strings no longer than X characters (16, IIRC) and contain no spaces. Basically, that the identifier as displayed within the H1 can also serve as the URL slug. So a section identifier like the one in this example is stored as `63.1-01throu`. In the case of Virginia, sometimes entire _sentences_ are stored in `desig`.

Basically it looks like Lexis did this thoughtlessly. There is no reliable source of an identifier.
